screens/PaymentInstructions.js

```diff
@@ -1,50 +1,52 @@
-import React, { useState, useEffect } from 'react'
+import React, { useState, useEffect } from "react";
+import { Text, StyleSheet, TextInput, Button, Image } from "react-native";
 import {
-  Text,
-  StyleSheet,
-  TextInput,
-  Button,
-  Image
-} from "react-native";
-import { TouchableOpacity, ScrollView, TouchableWithoutFeedback } from "react-native-gesture-handler";
+  TouchableOpacity,
+  ScrollView,
+  TouchableWithoutFeedback,
+} from "react-native-gesture-handler";
 import * as MailComposer from "expo-mail-composer";
 import * as ImagePicker from "expo-image-picker";
 import Constants from "expo-constants";
 import * as Permissions from "expo-permissions";
-import { SafeAreaView } from 'react-native-safe-area-context';
-import { cloudinaryPost } from './apiCalls'
+import { SafeAreaView } from "react-native-safe-area-context";
+import { cloudinaryPost } from "./apiCalls";
 // import { preventAutoHide } from 'expo/build/launch/SplashScreen';
 
-
 const PaymentInstructions = () => {
-  const [emailObject, updateEmailObject] = useState({})
-  const [image, uploadImage] = useState(null)
-  const [imageObj, setImageObj] = useState(null)
-  
+  const [emailObject, updateEmailObject] = useState({});
+  const [image, uploadImage] = useState(null);
+  const [imageObj, setImageObj] = useState(null);
+
   useEffect(() => {
     getPermissionAsync();
-  })
-  
+  });
+
   const submitHandler = async () => {
-    const imageUrl = await cloudinaryPost(imageObj)
+    const imageUrl = await cloudinaryPost(imageObj);
     MailComposer.composeAsync({
```

Comment:
When I wrote the mock email initially I wrote from the POV of Auctionable Change but maybe we can adjust to be in the voice of the buyer